src/Sylius/Behat/Resources/config/suites/ui/inventory/inventory.yml

```diff
@@ -0,0 +1,33 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        ui_inventory:
+            contexts_as_services:
+                - sylius.behat.context.hook.doctrine_orm
+
+                - sylius.behat.context.setup.channel
+                - sylius.behat.context.setup.currency
+                - sylius.behat.context.setup.geographical
+                - sylius.behat.context.setup.payment
+                - sylius.behat.context.setup.product
+                - sylius.behat.context.setup.shipping
```

Comment:
Also I didn't find any shipping context usage. Please, can you check do you really need all of these context's?